.github/workflows/cicd_pipeline_golang.yaml

```
@@ -232,3 +234,12 @@ jobs:
           additional-tag: rsrv
           dockerRepo: ${{ secrets.rsrv-repo }}
           k8s-context: ${{ secrets.rsrv-k8s-context }}
+  check-cmdb:
+    if: ${{ github.ref_name == 'develop'}}
+    runs-on: [ self-hosted, linux, docker, kubectl, prod ]
```

```suggestion
    runs-on: [ self-hosted, linux, dev ]
```
Do we really need to run it on actions runner with access to prod clusters? We 3 prod runners and 9 - dev. Actuall we use "prod" runner only to get access to prof cluster (deploy to prod or reserve)

---

because the network access has been provided to prod cluster from only prod runner, that's why I have to use prod runner here. But I can eliminate docker and kubectl tags